tests/vm_test.sh

```diff
@@ -42,38 +41,38 @@ function test_vm_mount()
     return
   }
 
-  ID=1
-  output=$(vm_mount "TEST_MODE")
+  output=$(
+    function findmnt()
+    {
+      echo "anything"
+    }
+    vm_mount "TEST_MODE"
+  )
```

Comment:
Niceeeee!!! Could you make a doc patch under `test.rst` that explains this technique?